app/assets/javascripts/discourse/tests/unit/models/user-test.js

```diff
@@ -70,54 +68,6 @@ module("Unit | Model | user", function () {
     );
   });
 
-  test("resolvedTimezone", function (assert) {
```

Comment:
It would be great if the timezone-guessing logic in `createCurrent()` was tested (like it was in this removed test)

Reply:
Yeah, that's an important thing to test. At first glance, I thought it'd be tough to test it, but turned out it's pretty easy. I've added tests.